src/services/reporting.js

```diff
@@ -24,3 +24,31 @@ export const getReports = async (request, from, to, intervals) => {
     throw error
   }
 }
+
+export const getLastSent = async (request) => {
+  try {
+    const { payload } = await wreck.get(`${baseUrl}/last-sent`, {
+      headers: { authorization },
+      json: 'strict'
+    })
+
+    return payload
+  } catch (error) {
+    request.logger.setBindings({ error })
+    throw error
+  }
+}
+
+export const getLastReceived = async (request) => {
+  try {
+    const { payload } = await wreck.get(`${baseUrl}/last-received`, {
+      headers: { authorization },
+      json: 'strict'
+    })
+
+    return payload
+  } catch (error) {
+    request.logger.setBindings({ error })
+    throw error
+  }
+}
```

Comment:
As per the above comment the export can be removed from these functions and a single getLatestActivity function can be exported here which invokes getLastReceived and getLastSent